modules/ztp-querying-the-policy-compliance-status-for-each-cluster.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-querying-the-policy-compliance-status-for-each-cluster_{context}"]
+= Querying the policy compliance status for each cluster
+
+After you have created the validator inform policies for your clusters and pushed them to the
+the zero touch provisioning (ZTP) Git repository, you can check the status of each cluster for policy compliance.
+
+.Procedure
+
+. To query the status of the spoke clusters, use either the {rh-rhacm-first} web console or the CLI:
++
+* To query status from the {rh-rhacm} web console, perform the following actions:
++
+.. Click *Governance* -> *Find policies*.
+.. Search for *du-validator-policy*
+.. Click into the policy.
+
+* To query status using the CLI, run the following command:
++
+[source,terminal]
+----
+$ oc get policies du-validator-policy -n <namespace_for_common> -o jsonpath={'.status.status'} | jq
```

Comment:
This should be changed to following to align with other part of the docs:

oc get policies -n <namespace_for_group> <validator_policy_name> -o jsonpath={'.status.status'} | jq